### PR TITLE
Mirriam development

### PR DESCRIPTION
**Pull request by mirriamchemu**
I am almost done with my portfolio, please take a look.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,124 @@
...
+      <h1>My Background</h1>
+      <p> My full name is Mirriam Chemutai Ronoh.
+          I was born and raised in Kericho County, Kenya. A little bit about 
+          my names; my grandmother gave me the name Mirriam, it's a Biblical name,
+          though it's spelt Miriam in the Bible. In my tribe (Kipsigis), middle names
+          are given with respect to the time of birth. I was named Chemutai because
+          I was born at dawn. 'Mutai' means dawn in my tribe and 'Che' is a girl's
+          name's prefix. Ronoh is my surname. I went to an all girls' 
+          boarding school, Alliance Girls' High School,
+           in a small town called Kikuyu, which is about
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/index.html]: The spacing looks off on these few lines.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,124 @@
...
+          still undecided on what I wanted to pursue, and after taking some 
+          Computer Science classes, I declared a Computer and Information Science
+          major after my freshman year. I also plan to minor in Mathematics.
+          Looking ahead, I am still quite unsure about what I want to do after graduation.
+          I am interested in pursuing a Masters or a pHD program in Machine
+          Learning. Artficial Intelligece fascinates me and I would love to be 
+          part of the commmunity that is trying to positively change various aspects
+          of life through computer technology. 
+          </p>
+          <p>During me free time, I like to stay indoors. Mostly, I like putting on
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/index.html]: nit: During my free time.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,124 @@
...
+          I am interested in pursuing a Masters or a pHD program in Machine
+          Learning. Artficial Intelligece fascinates me and I would love to be 
+          part of the commmunity that is trying to positively change various aspects
+          of life through computer technology. 
+          </p>
+          <p>During me free time, I like to stay indoors. Mostly, I like putting on 
+              headphones and clean the house, organize my space or cook something I like.
+               On a
+              Friday night, you'd likely find me keeping up with a show on Netflix, going 
+              to catch a movie at the theatre or just sleeping. My weekends are mostly
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/index.html]: spacing looks off here.

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -8,11 +8,124 @@
...
+              to register by assigning each user a distinct ID and username. Some users
+              can create groups (channels), each with its name, where one user is the owner who admits
+              other members and can also remove members. Users can send send messages
+              to each other and also to channels.
+          </p>
+
+          <p>(a)Hashing and Tries: Implemented a working HashMap using a resizing linked
+              list. Every new item inserted into the map is assigned a location 
+              with respect to its key hashcode. 
+               </p>
```
Comment (kmontgomery) [inline portfolio/src/main/webapp/index.html]: spacing looks off here.

**Comment by kmontgomery** [comment]
One other comment. Please update the name of the pull request to better describe what it is about.

**Comment by mirriamchemu** [comment]
Thanks for your comments. I'll get right into making the changes recommended.

**Pull request merged**